### PR TITLE
BZ2024153: Changing precopy interval

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=2024153

Change precopy interval with forkliftcontroller patch

Preview: https://deploy-preview-254--forklift-documentation.netlify.app/documentation/doc-migration_toolkit_for_virtualization/master/#changing-precopy-intervals_forklift

### CODE REVIEW

**Comment by apinnick** [comment]
@mguetta1  Ready for QE.

File: documentation/modules/changing-precopy-intervals.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="changing-precopy-intervals_{context}"]
+= Changing precopy intervals for warm migration
+
+The snapshots for the precopy stage of warm migration are created at 60-minute intervals by default.
+
+You can change the snapshot interval by patching the `forklift-controller` custom resource (CR).
```
Comment (mguetta1) [inline documentation/modules/changing-precopy-intervals.adoc:10]: ...ForkliftController custom resource (CR)
forklift-controller it is the name of the resource

File: documentation/modules/changing-precopy-intervals.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+.Procedure
+
+* Patch the `forklift-controller` CR:
```
Comment (mguetta1) [inline documentation/modules/changing-precopy-intervals.adoc:14]: ForkliftController

File: documentation/modules/changing-precopy-intervals.adoc [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
++
+[source,terminal]
+----
+$ oc patch forkliftcontroller forklift-controller -p '{"spec": {"controller_precopy_interval": <6>}}' --type=merge <1>
+----
```
Comment (mguetta1) [inline documentation/modules/changing-precopy-intervals.adoc:19]: ForkliftController is a namespace-scoped resource, so:
oc patch forkliftcontroller/forklift-controller -n openshift-mtv -p '{"spec": {"controller_precopy_interval": <60>}}' --type=merge

**Review by mguetta1 — CHANGES_REQUESTED** [review]

File: documentation/modules/changing-precopy-intervals.adoc (hunk 1 above)
Comment (apinnick) [reply documentation/modules/changing-precopy-intervals.adoc]: I fixed this alreadt.

File: documentation/modules/changing-precopy-intervals.adoc (hunk 2 above)
Comment (apinnick) [reply documentation/modules/changing-precopy-intervals.adoc]: already fixed

File: documentation/modules/changing-precopy-intervals.adoc (hunk 3 above)
Comment (apinnick) [reply documentation/modules/changing-precopy-intervals.adoc:17]: Done

**Pull request merged**